client/src/components/ResponseHandler.tsx

```diff
@@ -24,24 +24,42 @@ function ResponseHandler() {
     fetchData();
   }, []);
 
-  // Send `userInput` to the server, receive response, then add to `conversation`.
-  const handleUserInput = async (userInput: string) => {
+  useEffect(() => {
+    if (gameID) {
+      handleUserInput("");
+    }
+  }, [gameID]);
```

Comment:
This is called every time that `gameID` changes, right? Just want to make sure I understand how the `useEffect` works.